conf.py

```diff
@@ -35,7 +35,8 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store',
+                    'README.md', 'CONTRIBUTING.md', 'tools/README.md']
```

Comment:
Why is `CONTRIBUTING.md` excluded? Will it maybe be linked from a different place? It is useful for people who read the documentation to know how to contribute to it.